store/interfaces.json

```
@@ -2,16 +2,29 @@
   "condition-layout": {
     "component": "Noop"
   },
+  "condition-layout.product": {
+    "component": "ConditionLayoutProduct",
+    "composition": "children",
+    "content": {
+      "$ref": "#/definitions/ConditionLayoutProduct"
```

Use full path so if people extend this it won't break

```suggestion
      "$ref": "app:vtex.condition-layout#/definitions/ConditionLayoutProduct"
```